optimum/exporters/onnx/base.py

```diff
@@ -550,16 +550,19 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         input_names = [key for key in self.inputs.keys() if not key.startswith("past_key_values")]
         if self.use_past:
             input_names.append("past_key_values")
+
         for input_name in input_names:
             input_was_inserted = False
             for dummy_input_gen in dummy_inputs_generators:
                 if dummy_input_gen.supports_input(input_name):
-                    if self.use_past is True and input_name == "decoder_input_ids":
+                    # models from TextSeq2SeqOnnxConfig use decoder_input_ids as input name
+                    # while models from TextDecoderOnnxConfig use input_ids, hence the check for both
+                    if self.use_past is True and input_name in ["decoder_input_ids", "input_ids"]:
```

Comment:
I did not add `attention_mask` here because then we would need to distinguish between the seq2seq and causallm, which adds yet complexity in `OnnxConfigWithPast` which is not supposed to handle this. WDYT @michaelbenayoun ?